### DISCUSSION TITLE
Cue list

### DISCUSSION THREAD

**Discussion by JonathanM32** [jwetzell/qController#5] [General]
Hi. I'm not sure if this is a bug or something misconfigured on qlab but the cues do not show up in the app. The screen is stuck on loading workspace.

**Comment by jwetzell** [comment]
Can you give some context to your setup.
What QLab Version, App Version, Phone OS Version? Is the workspace password protected. Have you checked pinging the QLab computer from the phone to verify networking between phone and QLab?

**Comment by JonathanM32** [comment]
I am using v1.6.4 of qControl on Android 13 running on a Pixel 6. Confirmed
same behavior on an iPad. Qlab is v4.7, no password. qControl is able to
advance and play cues, it just doesn't show the cue list. Whole system is
running on university wifi which could be messing things up.
Appreciate your help

**Reply by jwetzell** [reply]
I actually am able to reproduce this with the version from the App Store. Fighting some expired developer certificates to be able to rebuild a new version. I will keep you updated when I get a fix out.

**Reply by JonathanM32** [reply]
Thank you!

**Reply by jwetzell** [reply]
I think I've found the bug I introduced when switching to a TCP Library. I reverted back to just standard .NET TCP and have begun the rollout on Android. Still trying to sort out my expired Dev/Distro certificates for Apple.
